_posts/en/newsletters/2024-08-30-newsletter.md

```diff
@@ -0,0 +1,122 @@
+  [covenant][topic covenants] functionality to Bitcoin scripts.
+
+- [Why do some bech32 bitcoin addresses contain a large number of 'q's?]({{bse}}123902)
+  Vojtěch Strnad reveals that the OLGA protocol encodes arbitrary data into
+  P2WSH outputs with part of the scheme requiring 0-padding (0 is encoded as 'q'
+  in [bech32][topic bech32]).
+
+- [How does a 0-conf signature bond work?]({{bse}}124022)
+  Matt Black outlines how funds locked in an OP_CAT-based covenant could provide
+  incentives for spenders to not [RBF][topic rbf] fee bump their transactions,
+  potentially increasing acceptance of zero-confirmation transactions.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Core Lightning 24.08rc2][] is a release candidate for the next major
+  version of this popular LN node implementation.
+
+- [LND v0.18.3-beta.rc1][] is a release candidate for a minor bug fix
+  release of this popular LN node implementation.
+
+- LDK 0.0.124-beta FIXME:harding
+
+- BDK 1.0.0-beta.2 FIXME:harding
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [LDK #3263][] simplifies how it handles [onion messages][topic onion messages]
+  responses by removing the message type bound from the `ResponseInstruction`
+  struct, and introducing a  new `MessageSendInstructions` enum based on the
+  updated `ResponseInstruction`, that can handle both [blinded][topic rv
+  routing] and non-blinded reply paths. The `send_onion_message` method now uses
+  `MessageSendInstructions`, allowing users to specify reply paths without
+  needing to figure out the pathfinding themselves. A new option,
+  `MessageSendInstructions::ForReply`, lets message handlers send responses
+  later without creating circular dependencies in the code. See Newsletter
+  [#303][news303 onion].
+
+- [LDK #3247][] deprecates the `AvailableBalances::balance_msat` method in favor
+  of the `ChannelMonitor::get_claimable_balances` method, which provides a more
+  straightforward and accurate approach to obtaining a channel's balance. The
+  deprecated method’s logic is now outdated as it was originally designed to
+  handle potential underflow issues when balances included pending HTLCs (those
+  that could later be reversed).
+
+- [BDK #1569][] adds the `bdk_core` crate to move some types over from
```

Comment:
This wording isnt clear to me. The introduction of the bdk_core crate has what relation to these other crates before/after this change?

Reply:
the types listed are taken from bdk_chain and moved into the new crate bdk_core.

the chain source crates depend on those types, so they now depend on bdk_core, instead of bdk_chain.